spring-integration-webflux/src/main/java/org/springframework/integration/webflux/outbound/WebFluxRequestExecutingMessageHandler.java

```diff
@@ -201,6 +201,12 @@ public void setPublisherElementTypeExpression(Expression publisherElementTypeExp
 		this.publisherElementTypeExpression = publisherElementTypeExpression;
 	}
 
+	/**
+	 * Configure expression to evaluate request attribute which will be added to webclient request attribute.
+	 * @param attributeVariablesExpression the expression to evaluate request attribute.
+	 * @since 6.0
+	 * @see WebFluxRequestExecutingMessageHandler#evaluateAttributeVariables(Message)
```

Comment:
You are pointing here to `private` method which is out of end-user interest.
I'd say `@see WebClient.RequestBodySpec#attributes`